library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -231,6 +232,23 @@ public void onMapClick(LatLng latLng) {
     polyline.removeFromGoogleMap();
   }
 
+  @Override
+  public void zoomToFitMarkers(int padding, AirMapMarker... markers) {
+    LatLngBounds.Builder b = new LatLngBounds.Builder();
+    for(AirMapMarker marker : markers){
+      b.include(marker.getLatLng());
+    }
+    LatLngBounds bounds = b.build();
+    final CameraUpdate cu = CameraUpdateFactory
+            .newLatLngBounds(bounds, padding);
+    googleMap.setOnMapLoadedCallback(new GoogleMap.OnMapLoadedCallback() {
```

Comment:
@petzel I need to figure out a different way to set this callback without overwriting any existing callbacks then because zoomToFitMarkers can be called after onMapInitialized but the MapView isn't inflated yet. This results in a force close with the following error:

"java.lang.IllegalStateException: Error using newLatLngBounds(LatLngBounds, int): Map size can't be 0"